Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "holeyvec"
+version = "0.1.0"
+edition = "2021"
+license-file = "LICENSE"
```

Comment:
You can also specify the license directly.  I don't know if that's better than specifying a license file.  But given it's a standard license it might help crates.io.

```
license = "MIT"
```

Reply:
I used separate file because SingularityNET Foundation is mentioned in it.